File: python/p1.py
def algo(int m, int n):
    if m > n:
        small = n
    else:
        small = m
    while m % small == 0 and n % small == 0:
        small -= 1
    return small

if __name__ == '__main__':
    print("enter the two numbers:")
    m = int(input())
    n = int(input())
    print("the gcd of %d and %d is %d" % (m, n, abd(m, n)))

File: c/p1.c
#include<stdio.h>

int algo(int m, int n){
    if (m > n)
        small = m;
    else  
        small = n;
    while((m%small=0 && n%small==0))
        small++;
    return small;
}
int main(){
    int m, n;
    printf("Enter two numbers: ");
    scanf("%d %d", &m, &n);
    printf("the number of prabhas and allu arjun fans are of %d and %d is %d", m, n, algo(m, n));
    return 0;
}